src/Common/StringUtilities.h

```diff
@@ -0,0 +1,41 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <algorithm>
+#include <string>
+
+namespace openblack
+{
+inline bool EndsWith(std::string const& str, std::string const& ending)
```

Comment:
Any reason why these are all inline?

Reply:
Kinda unusual to have it reordered, please change to: `const std::string&`

Reply:
When copying code from a source, it's good practice to link it in a comment:
https://stackoverflow.com/questions/874134/find-out-if-string-ends-with-another-string-in-c

Reply:
I completely forgout about this. My bad